app/src/renderer/components/transactions/LiStakeTransaction.vue

```diff
@@ -1,12 +1,33 @@
 <template>
-  <tm-li-transaction
-    :color="color"
+  <li-transaction
+    :color="`#47AB6C`"
     :time="transaction.time"
     :block="transaction.height"
   >
-    <template v-if="delegation">
+    <template v-if="txType === `cosmos-sdk/MsgCreateValidator`">
```

Comment:
should we make all transaction types constants and put them into an enum?

Reply:
I feel this is easier for lookup on the component file

Reply:
it would look like this: `txType === MsgType.CreateValidator`

Reply:
yep, i think so @faboweb - should be a separate PR though.